bundle/Dockerfile

```diff
@@ -130,10 +131,16 @@ COPY --from=build /go/src/github.com/plgd-dev/hub/cloud2cloud-connector/config.y
 COPY --from=build /go/src/github.com/plgd-dev/hub/bundle/run.sh /usr/local/bin/run.sh
 COPY --from=build /go/src/github.com/plgd-dev/hub/bundle/nginx /nginx
 
+# install scylla
+RUN curl -sSf get.scylladb.com/server | sudo bash -s -- --scylla-version 5.2
```

Comment:
Do we want to hard code to version `5.2`?

Reply:
Yes. We want to update scylla manually - we don't know if the newest version is compatible with the bundle scylla configuration.